aether-kernel/sql/create_readonly_user.py

```diff
@@ -59,24 +66,33 @@ def main():
     dbname = os.environ['DB_NAME']
     host = os.environ['PGHOST']
     port = os.environ['PGPORT']
-    user = os.environ['PGUSER']
+    root_user = os.environ['PGUSER']
+    root_password = os.environ['PGPASSWORD']
 
+    logger.debug('db://{user}:{pwrd}@{host}:{port}/{dbname}'.format(
+        user=root_user,
+        pwrd=(len(root_password) * '*'),
+        host=host,
+        port=port,
+        dbname=dbname,
+    ))
     postgres_credentials = {
         'dbname': dbname,
         'host': host,
         'port': port,
-        'user': user,
+        'user': root_user,
+        'password': root_password,
     }
 
     with psycopg2.connect(**postgres_credentials) as conn:
-        role = os.environ['KERNEL_READONLY_DB_USERNAME']
-        password = os.environ['KERNEL_READONLY_DB_PASSWORD']
+        ro_user = os.environ['KERNEL_READONLY_DB_USERNAME']
+        ro_password = os.environ['KERNEL_READONLY_DB_PASSWORD']
         cursor = conn.cursor()
         query = sql.SQL(CREATE_READONLY_USER).format(
             database=sql.Identifier(dbname),
-            role=sql.Identifier(role),
-            rolename=sql.Literal(role),
-            password=sql.Literal(password),
+            role=sql.Identifier(ro_user),
+            rolename=sql.Literal(ro_user),
```

Comment:
Because you are already refactoring this... could you remove the `{rolename}` option? (being the same as `{role}` it's just error-prone) :grimacing: